tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/wizard/NewLiferayPluginProjectWizard.java

```diff
@@ -154,16 +161,41 @@ protected void performPostFinish()
     {
         super.performPostFinish();
 
+        List<IProject> finalProjects = new ArrayList<IProject>();
+
         final NewLiferayPluginProjectOp op = element().nearest( NewLiferayPluginProjectOp.class );
+
         final IProject project = CoreUtil.getProject( op.getFinalProjectName().content() );
 
-        try
+        if( ProjectUtil.isMavenProject( project ) )
```

Comment:
We should not put maven specific code in the project.ui plugin.  We need to find a way to solve this without resorting to a check for maven here.